src/main/java/com/ioet/bpm/skills/boundaries/SkillController.java

```diff
@@ -33,6 +34,12 @@ public SkillController(SkillRepository skillRepository) {
                 .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
     }
 
+    @GetMapping("/search/{name}")
```

Comment:
We don't want to add `search` as part of the URI. Let remove it in this way:

@GetMapping("/")

Reply:
I used `search` because when I built the method to search for the name it generated an error of code duplication.